Website/src/items/content/EdgeWebsite/Data/CMP Data/Testimonies/Alba.yml

```diff
@@ -33,7 +36,7 @@ Languages:
         default\Anonymous
     - ID: "8cdc337e-a112-42fb-bbb4-4143751e123f"
       Hint: __Revision
-      Value: "4000c87f-b106-42a5-b135-ea35b776f13f"
+      Value: "85203702-a7ae-41ef-9b52-513ede880168"
     - ID: "a4eb8a28-879d-41d9-87c9-2ed2f03d0ae4"
       Hint: Logo
```

Comment:
The DemoEdge URL in this Logo field is not valid anymore. Can you please update it to a playsummit URL?